test/index.test.js

```diff
@@ -0,0 +1,7 @@
+var Enzyme = require("enzyme");
+var Adapter = require("enzyme-adapter-react-16");
+
+Enzyme.configure({ adapter: new Adapter() });
+
+var testsContext = require.context(".", true, /.test.js$/);
+testsContext.keys().forEach(testsContext);
```

Comment:
This is boilerplate that allows us to bundle all of our tests using webpack, without creating a separate bundle per file.